src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -64,6 +64,8 @@ internal sealed class MeterProviderSdk : MeterProvider
 
         OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent("Building MeterProvider.");
 
+        OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Metric overflow attribute key set to: {isEmitOverflowAttributeKeySet}");
```

Comment:
a more user friendly message would "Experimental feature "metric overflow attribute" is enabled/disabled".

The current msg could be interpreted as "key" used for overflow is set to true/false instead of "otel.metric.overflow". 🤭

Reply:
The intended audience for verbose logging is us so it should be fine 😀 I agree that we could still make it more user-friendly.